src/app/widgets/buses/admin-panel-buses/admin-panel-buses.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from "@angular/core";
+import { FormControl, FormGroup, Validators } from "@angular/forms";
+import { Router } from "@angular/router";
+import { busStopNames } from "../model-buses/busStop.model";
+import { FetchBusStopsService } from "../services-buses/fetch-bus-stops.service";
+import { LocalBusStorageService } from "../services-buses/local-bus-storage.service";
+import { SelectedBusStopService } from "../services-buses/selected-bus-stop.service";
+
+@Component({
+  selector: "app-admin-panel-buses",
+  templateUrl: "./admin-panel-buses.component.html",
+  styleUrls: ["./admin-panel-buses.component.scss"],
+})
+export class AdminPanelBusesComponent implements OnInit {
+  busStops: busStopNames = [];
+
+  defaultBusStopForm!: FormGroup;
+
+  constructor(
+    private router: Router,
+    private fetchBusStopsService: FetchBusStopsService,
+    private selectedBusStopService: SelectedBusStopService,
+    private localBusStorageService: LocalBusStorageService
+  ) {}
+
+  ngOnInit(): void {
+    this.fetchBusStopsService
+      .fetchBusStops()
+      .subscribe((busStops) => (this.busStops = Object.values(busStops)));
+
+    this.defaultBusStopForm = new FormGroup({
+      defaultBusStop: new FormControl(null, Validators.required),
```

Comment:
Przydała by się zaznaczona pokazania zaznaczonej wartości gdy przejdziemy do admina jeszcze raz.
Aby to zrobić w new FormControl zamiast null powinieneś sprawdzać czy istnieje w localStorage  wpis i jeśli tak to go dodać zamiast null jeśli nie to null